app/models/complaint.rb

```diff
@@ -75,7 +75,7 @@ class Complaint < ActiveRecord::Base
     'school state' => { column: :state, converter: BaseConverter },
     'submitted' => { column: :submitted, converter: BaseConverter },
     'closed' => { column: :closed, converter: BaseConverter },
-    'closed reason' => { column: :closed_reason, converter: BaseConverter },
+    'closed reason' => { column: :closed_reason, converter: DowncaseConverter },
```

Comment:
No, it shouldn't. I think that the exported data should represent exactly what is in the database at the time of export, with the same formatting and casing.